core/store/src/trie/trie_storage.rs

```diff
@@ -369,6 +381,16 @@ pub struct TrieCachingStorage {
     pub(crate) db_read_nodes: Cell<u64>,
     /// Counts trie nodes retrieved from the chunk cache.
     pub(crate) mem_read_nodes: Cell<u64>,
+
+    // Store counteres here to avoid overhead of looking them up on hot paths.
```

Comment:
```suggestion
    // Counters tracking events happening while querying shard cache. Stored here to avoid overhead of looking them up on hot paths.
```